spec/client/ics-007-tendermint-client/README.md

```diff
@@ -290,178 +366,47 @@ Tendermint client state verification functions check a Merkle proof against a pr
   clientState: ClientState,
   height: Height,
-  delayPeriodTime: uint64,
-  delayPeriodBlocks: uint64,
-  prefix: CommitmentPrefix,
+  delayTimePeriod: uint64,
+  delayBlockPeriod: uint64,
   proof: CommitmentProof,
-  portIdentifier: Identifier,
-  channelIdentifier: Identifier,
-  sequence: uint64,
-  acknowledgement: bytes) {
-    path = applyPrefix(prefix, "ports/{portIdentifier}/channels/{channelIdentifier}/acknowledgements/{sequence}")
+  path: CommitmentPath,
+  value: []byte) {
     // check that the client is at a sufficient height
     assert(clientState.latestHeight >= height)
     // check that the client is unfrozen or frozen at a higher height
     assert(clientState.frozenHeight === null || clientState.frozenHeight > height)
-    // fetch the processed time
-    processedTime = get("clients/{identifier}/processedTimes/{height}")
-    // fetch the processed height
-    processedHeight = get("clients/{identifier}/processedHeights/{height}")
     // assert that enough time has elapsed
     assert(currentTimestamp() >= processedTime + delayPeriodTime)
     // assert that enough blocks have elapsed
     assert(currentHeight() >= processedHeight + delayPeriodBlocks)
     // fetch the previously verified commitment root & verify membership
     root = get("clients/{identifier}/consensusStates/{height}")
-    // verify that the provided acknowledgement has been stored
-    assert(root.verifyMembership(clientState.proofSpecs, path, hash(acknowledgement), proof))
-}
-
-function verifyPacketReceiptAbsence(
-  clientState: ClientState,
-  height: Height,
-  delayPeriodTime: uint64,
-  delayPeriodBlocks: uint64,
-  prefix: CommitmentPrefix,
-  proof: CommitmentProof,
-  portIdentifier: Identifier,
-  channelIdentifier: Identifier,
-  sequence: uint64) {
-    path = applyPrefix(prefix, "ports/{portIdentifier}/channels/{channelIdentifier}/receipts/{sequence}")
-    // check that the client is at a sufficient height
-    assert(clientState.latestHeight >= height)
-    // check that the client is unfrozen or frozen at a higher height
-    assert(clientState.frozenHeight === null || clientState.frozenHeight > height)
-    // fetch the processed time
-    processedTime = get("clients/{identifier}/processedTimes/{height}")
-    // fetch the processed height
-    processedHeight = get("clients/{identifier}/processedHeights/{height}")
-    // assert that enough time has elapsed
-    assert(currentTimestamp() >= processedTime + delayPeriodTime)
-    // assert that enough blocks have elapsed
-    assert(currentHeight() >= processedHeight + delayPeriodBlocks)
-    // fetch the previously verified commitment root & verify membership
-    root = get("clients/{identifier}/consensusStates/{height}")
-    // verify that no acknowledgement has been stored
-    assert(root.verifyNonMembership(clientState.proofSpecs, path, proof))
+    // verify that the provided consensus state has been stored
+    assert(root.verifyMembership(path, value, proof))
```

Comment:
Cf. [ICS-23](https://github.com/cosmos/ibc/blob/main/spec/core/ics-023-vector-commitments/README.md#proof-verification), `verifyMembership` has the following signature:
```typescript
type verifyMembership = (root: CommitmentRoot, proof: CommitmentProof, path: CommitmentPath, value: Value) => boolean
```